src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const arrFromSource = sourceString
+    .trim()
+    .split(';')
+    .map((el) =>
+      el
+        .trim()
+        .split(':')
+        .map((item) => item.trim()),);
+
+  const obj = {};
 
+  arrFromSource.forEach((el) => {
+    if (el.length === 2) {
+      obj[el[0]] = el[1];
```

Comment:
Instead of using a `forEach` loop to build the object, consider using `reduce` which is more in line with functional programming practices and avoids the need to declare an empty object upfront.